### ISSUE TITLE
Possible false-positive

### ISSUE THREAD

**Issue by Naramsim** [Clever/csvlint#45]
Hi, when scanning this file: https://github.com/PokeAPI/pokeapi/blob/master/data/v2/csv/pokemon_species_flavor_text.csv

your tool (both `v0.3.0` and `master`) gives me the following output:

```log
Record #62175 has error: bare " in non-quoted-field

unable to parse any further
```

I checked line 62175 and it's valid. Upon isolating it you tool as well validates it. I don't understand why i get this error. I scanned the entire file with a custom bash script and found no issues. 

Thanks!

**Comment by n0rtr0n** [comment]
Hi @Naramsim, thank you for asking. While there have been some recent updates to standardize some (mostly infrastructural) patterns across all of our repos, we are otherwise not actively maintaining this project right at this moment.
